src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
The number '40' is a magic number. Consider defining it as a constant on top of your function, giving it a descriptive name.